first_databank/models/fdb_img_date.py

```diff
@@ -2,12 +2,41 @@
 # © 2015-TODAY LasLabs Inc.
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 
-from openerp import models, fields
+from datetime import datetime
+
+from openerp import models, fields, api
 
 
 class FdbImgDate(models.Model):
     _name = 'fdb.img.date'
     _description = 'Fdb Img Date'
-    start_date = fields.Datetime()
+    _inherits = {'fdb.img': 'image_id'}
+
+    start_date = fields.Datetime(
+        required=True,
+    )
     stop_date = fields.Datetime()
-    img_id = fields.Integer()
+    image_id = fields.Many2one(
+        string='Image',
+        comodel_name='fdb.img',
+        required=True,
+        ondelete='restrict',
+    )
+    active = fields.Boolean(
+        compute='_compute_active',
+    )
+    relation_id = fields.Many2one(
```

Comment:
Ew, I don't like that either. Ummm, ```image_relation_id``` is the best I have, because it's slightly more descriptive but it still sucks